batch/batch/driver/instance_pool.py

```diff
@@ -277,7 +277,7 @@ def add_instance(self, instance):
 NAME=$(curl -s http://metadata.google.internal/computeMetadata/v1/instance/name -H 'Metadata-Flavor: Google')
 
 # this has to match LogStore.worker_log_path
-gsutil -m cp run.log worker.log /var/log/syslog gs://$BUCKET_NAME/batch2/logs/$INSTANCE_ID/worker/$NAME/
+gsutil -m cp run.log worker.log /var/log/syslog gs://$BUCKET_NAME/batch/logs/$INSTANCE_ID/worker/$NAME/
```

Comment:
I can't make the comment in the right place. `batch2-agent` tag needs to be changed to match the build step `delete-batch_instances`